tests/parsing/grammar/expressions/expected/record.res.txt

```diff
@@ -2,8 +2,8 @@ let r = { a = expr }
 let r = { a = expr }
 let r = { Parsetree.pexp_attributes = [||]; Parsetree.loc = loc }
 let r = { a; b; c }
-let r = { Parsetree.pexp_attributes; Parsetree.loc }
-let r = { Parsetree.pexp_attributes; Parsetree.loc }
+let r = { Parsetree.pexp_attributes = pexp_attributes; Parsetree.loc = loc }
```

Comment:
I think this shouldn't have changed?

Reply:
I think the AST is correct and that's just the way it is printed if "key != value expression".

Reply:
This change is indeed correct, the diff shows the correctly parsed AST.